### PR TITLE
Add Hub Hero to list of commercial JupyterHub providers

### PR DESCRIPTION
**Pull request by betatim**
Hello all!

I could not find the criteria you used to decide which commercial JupyterHub providers to include in the book. I operate https://hubhero.net which provides hubs for teaching professionals and would like to include it in the book.

I clearly have my own interests at mind here and can't make an objective judgement of how appropriate the text and inclusion is -> if you think some of it reads too much like a shameless advert with no substance please suggest how to change it.

### CODE REVIEW

**Comment by betatim** [comment]
The travis failure seems unrelated to this change :-/ https://travis-ci.com/jupyter4edu/jupyter-edu-book/builds/94312332#L397-L406

File: 06-getting-your-class-going-with-jupyter.md [hunk 1]
```diff
@@ -324,6 +324,15 @@ which provide free trials or a "freemium" pricing model. They include:
     like a way to create a course, invite students, and deploy
     auto-graded assignments.
 
+* HubHero ([https://hubhero.net](https://hubhero.net)) provides dependable
+  JupyterHub servers for teaching professionals. For courses of up to about 30
```
Comment (dsblank) [inline 06-getting-your-class-going-with-jupyter.md]:
```suggestion
  JupyterHub servers for teachers. For courses of up to about 30
```

File: 06-getting-your-class-going-with-jupyter.md [hunk 2]
```diff
@@ -324,6 +324,15 @@ which provide free trials or a "freemium" pricing model. They include:
...
+  students we offer [the community install](https://hubhero.net/community/) which
```
Comment (dsblank) [inline 06-getting-your-class-going-with-jupyter.md]:
```suggestion
  students they offer [the community install](https://hubhero.net/community/) which
```

**Comment by dsblank** [comment]
I think Travis CI went through some changes, and needed re-authorization. Let's see if it builds now....

**Comment by betatim** [comment]
Thanks for the suggestions and fixing travis!

File: 06-getting-your-class-going-with-jupyter.md [hunk 3]
```diff
@@ -324,6 +324,15 @@ which provide free trials or a "freemium" pricing model. They include:
     like a way to create a course, invite students, and deploy
     auto-graded assignments.
 
+* HubHero ([https://hubhero.net](https://hubhero.net)) provides dependable
```
Comment (labarba) [inline 06-getting-your-class-going-with-jupyter.md]: Since we, the book authors, haven't tried the service, we should avoid assessments like "dependable."

**Comment by dsblank** [comment]
Thanks all!

**Pull request merged**